coderd/database/dbauthz/dbauthz.go

```diff
@@ -2507,6 +2530,13 @@ func (q *querier) InsertTemplateVersionVariable(ctx context.Context, arg databas
 	return q.db.InsertTemplateVersionVariable(ctx, arg)
 }
 
+func (q *querier) InsertTemplateVersionWorkspaceTag(ctx context.Context, arg database.InsertTemplateVersionWorkspaceTagParams) (database.TemplateVersionWorkspaceTag, error) {
+	if err := q.authorizeContext(ctx, policy.ActionCreate, rbac.ResourceSystem); err != nil {
+		return database.TemplateVersionWorkspaceTag{}, err
+	}
+	return q.db.InsertTemplateVersionWorkspaceTag(ctx, arg)
+}
+
```

Comment:
I'm hesitant to just stuff this into the 'system' category. Given that the newly introduced table is related to template versions, `ResourceTemplate` would seem more appropriate.

Reply:
I can take another look, but I'm quite sure I aligned (stole) it with Template Variables.

Reply:
In that case, in order to avoid blocking, I'm fine with addressing this as a follow-up. Filed https://github.com/coder/coder/issues/13315